dts/bindings/timer/st,stm32-timers.yaml

```
@@ -46,3 +46,11 @@ properties:
                                                down.
 
       If absent, then STM32_TIM_COUNTERMODE_UP is used (reset state).
+
+  st,deadtime:
+    type: int
+    default: 0
+    description: |
+      Sets the deadtime configuration the Break and Dead-Time register for
```

Prefer `dead-time` to better match reference manual.
```suggestion
      Sets the dead-time configuration in the Break and Dead-Time register (BDTR) for
```
Could add:
 ```c
      Range [0 .. 255]